integration-tests/pom.xml

```diff
@@ -446,61 +450,7 @@
-                    <plugin>
-                        <groupId>de.thetaphi</groupId>
-                        <artifactId>forbiddenapis</artifactId>
-                        <configuration>
-                            <signaturesFiles>
-                                <!-- Needed because of https://github.com/policeman-tools/forbidden-apis/issues/126 -->
-                                <signaturesFile>../codestyle/joda-time-forbidden-apis.txt</signaturesFile>
-                                <signaturesFile>../codestyle/druid-forbidden-apis.txt</signaturesFile>
-                            </signaturesFiles>
-                        </configuration>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
-        <profile>
-            <id>int-tests-config-file</id>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.apache.maven.plugins</groupId>
-                        <artifactId>maven-failsafe-plugin</artifactId>
-                        <executions>
-                            <execution>
-                                <id>integration-tests</id>
-                                <phase>integration-test</phase>
-                                <goals>
-                                    <goal>integration-test</goal>
-                                    <goal>verify</goal>
-                                </goals>
-                            </execution>
-                        </executions>
-                        <configuration>
-                            <properties>
-                                <property>
-                                    <name>testrunfactory</name>
-                                    <value>org.testng.DruidTestRunnerFactory</value>
-                                </property>
-                            </properties>
-                            <argLine>
-                                -Duser.timezone=UTC
-                                -Dfile.encoding=UTF-8
-                                -Ddruid.test.config.type=configFile
-                                -Ddruid.test.config.configFile=${env.CONFIG_FILE}
-                                -Ddruid.client.https.trustStorePath=client_tls/truststore.jks
-                                -Ddruid.client.https.trustStorePassword=druid123
-                                -Ddruid.client.https.keyStorePath=client_tls/client.jks
-                                -Ddruid.client.https.certAlias=druid
-                                -Ddruid.client.https.keyManagerPassword=druid123
-                                -Ddruid.client.https.keyStorePassword=druid123
-                            </argLine>
-                            <suiteXmlFiles>
-                                <suiteXmlFile>src/test/resources/testng.xml</suiteXmlFile>
-                            </suiteXmlFiles>
-                        </configuration>
-                    </plugin>
+
```

Comment:
I dont understand the reason of this Profile. We can run tests with integration-tests profile and add 3 arguments: "skip.start.docker=true" "skip.stop.docker=true" "skip.build.docker=true" to get the same behaviour as with "int-tests-config-file" profile

Reply:
With the integration-tests profile, the test will run against docker cluster (tests will try to talk to env.DOCKER_IP). Those 3 arguments: "skip.start.docker=true" "skip.stop.docker=true" "skip.build.docker=true" does not change this fact. Whatever you set those argument to, using the integration-tests profile, tests will still talk to env.DOCKER_IP. Hence, if you do have skip.start.docker=true or skip.build.docker=true, you will need to make sure that somehow there is docker running at env.DOCKER_IP. 

On the other hand, the int-tests-config-file will not assume docker is running at all. It will ignore those 3 arguments: "skip.start.docker=true" "skip.stop.docker=true" "skip.build.docker=true" and also ignore env.DOCKER_IP. It will read a file located at env.CONFIG_FILE for the ip of druid nodes. You can provide the ips of the druid node and other dependency services. Those services can be running on cloud, or locally, or on your network, etc.  You can even use this profile and set the config file to point to your production cluster (not recommended though haha)

Reply:
I dont know how is possible to run docker containers on any IP because we have hard coded ip addresses in shell scripts that expect some container in some IP address
Anyway I will roll back this profile

Reply:
rolled back

Reply:
I mean that this profile, int-tests-config-file, is not for running integration tests with Docker. It can run integration tests against Druid running at the given ip. The Druid that is running at the given ip does not need to be run using Docker.